File: SurveyApp/mobileApp/lib/Modals/SubmissionValueInfo.dart
class SubmissionValueInfo{
  int Id;
int SubmissionId;
String QuestionText;
String AnswerText;

  Map<String, dynamic> toJson() {
    return {
      'Id': Id,
      'SubmissionId': SubmissionId,
      'QuestionText': QuestionText,
      'AnswerText': AnswerText
    };
  }
}

File: SurveyApp/Backend/Modals/SubmissionValueInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SurveyBeta.Modals
{
    public class SubmissionValueInfo
    {
        public int Id { get; set; }
        public int SubmissionId { get; set; }
        public string QuestionText { get; set; }
        public string AnswerText { get; set; }
    }
}